silkworm/silkrpc/common/block_cache.hpp

```diff
@@ -19,11 +19,12 @@
 #include <cstddef>
 #include <mutex>
 
-#include <boost/compute/detail/lru_cache.hpp>
+#include <boost/optional.hpp>
```

Comment:
You probably need to remove this include and add standard <optional> instead.

Reply:
ok